src/sql/workbench/browser/editor/profiler/dashboardInput.ts

```diff
@@ -93,6 +94,29 @@ export class DashboardInput extends EditorInput {
 		return name;
 	}
 
+	public override getTitle(verbosity?: Verbosity): string {
+		let connName = this.connectionProfile.connectionName ? (' (' + this.connectionProfile.connectionName + ')') : '';
+		let baseName = this.connectionProfile.serverName;
+		if (this.connectionProfile.databaseName && !this.isMasterMssql()) {
+			// Only add DB name if this is a non-default, non-master connection and if there is no user set profile name.
+			baseName = baseName + ':' + this.connectionProfile.databaseName;
+		}
+		let advancedOptions = this._connectionService.getNonDefaultOptions(this.connectionProfile);
+		advancedOptions = advancedOptions.replace('(', '[').replace(')', ']');
+
+		let fullTitle = baseName + connName + advancedOptions;
+
+		switch (verbosity) {
+			case Verbosity.SHORT:
+				return this.getName();
+			case Verbosity.LONG:
+				return fullTitle;
+			default:
+			case Verbosity.MEDIUM:
+				return fullTitle;
```

Comment:
What would be the desire to use Medium v/s Long? I see you're returning same title.
Can you add a doc to the method to explain with sample titles the method will return with verbosity set.